src/pysoundcomparisons/__main__.py

```diff
@@ -358,6 +375,26 @@ def create_offline_version(args):
     _copy_path(os.path.join(sndCompRepoPath, "LICENSE"), outPath)
     _copy_path(os.path.join(sndCompRepoPath, "README.md"), outPath)
 
+    # Download all contributor images hosted on CDSTAR
+    args.log.info("downloading images from CDSTAR ...")
+    catalog = _get_catalog(args, 'imagefiles')
+    for obj in catalog:
+        md = obj.metadata
+        if md['name']:
+            response = None
+            try:
+                response = urlopen("http://cdstar.shh.mpg.de/bitstreams/%s/%s" % (obj.id, md['path']))
+            except Exception as e:
+                shutil.rmtree(outPath)
+                args.log.error("Error while downloading image file %s\n%s" % (md['path'], e))
+                return
+            if not response:
```

Comment:
The way I read the docs, `urlopen` will never return `None`. So this branch will never be executed.